example/handwritten-src/swift/TXSTextboxListenerDebugableImpl.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TXSTextboxListenerImpl.swift
+//  TextSort
+//
+//  Created by Bruno Coelho on 10/03/2017.
+//  Copyright © 2017 Dropbox, Inc. All rights reserved.
+//
+
+import UIKit
+
+@objc class TXSTextboxListenerDebugableImpl : NSObject, TXSTextboxListener {
+    private var textView_: UITextView
```

Comment:
trailing underscores are not idiomatic in Swift, i'd recommend renaming to `textView` (and you'd do `self.textView = textView` on line 16 below